otx/algorithms/action/configs/classification/movinet/model.py

```diff
@@ -0,0 +1,128 @@
+"""Model configuration of MoViNet model for Action Classification Task."""
+
+# Copyright (C) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+# pylint: disable=invalid-name
+
+num_classes = 400
+model = dict(
+    type='Recognizer3D',
+    backbone=dict(
+        type='MoViNetBase',
+        name="MoViNetA0",
+        num_classes=num_classes),
+    cls_head=dict(
+        type='MoViNetHead',
+        in_channels=480,  # A0: 480, A1: 600, A2: 640, A3: 744, A4: 856, A5: 992
+        hidden_dim=2048,
+        num_classes=num_classes,
+        spatial_type='avg',
+        dropout_ratio=0.5),
+    # model training and testing settings
+    train_cfg=None,
+    test_cfg=dict(average_clips='prob'))
+
+
+# dataset settings
+dataset_type = "RawframeDataset"
+img_norm_cfg = dict(mean=[0.0, 0.0, 0.0], std=[255.0, 255.0, 255.0], to_bgr=False)
+
+clip_len = 8
+frame_interval = 4
+train_pipeline = [
```

Comment:
I wanna suggest to separate the data pipeline file from model.py

Reply:
Thank you for suggestion, but that's not in my PR's scope. @jaegukhyun and I will create another PR to resolve that.

Reply:
I agree with Jihawn, after adding all cli task of action we'll decide template structure

Reply:
Good

Reply:
Sorry but I'm not sure why this should be done in another PR. Is it currently inoperable if separated into data_pipeline.py?

Reply:
It can be divided into data_pipeline.py, model.py, recipe.py and so on. However we haven't decide how we divide the total model.py. In addition I think the loading fixed data_pipeline.py strategy should be changed. If we support more learning methods. Data augmentation of semi-sl, self-sl, and normal finetuning should be different.

Reply:
The current general ((cls, det, seg tasks) structure use {model.py + data_pipeline.py}. If we are going to take the action task separately, I agree to leave it as it is and refactor it separately, but if we are going to follow a general structure, it seems appropriate to fit it into the current structure. what do you think?

Reply:
@JihwanEom Let's split data_pipeline.py from model.py. You need to change X3D's config. There is no need to differentiate data pipeline's of the two model. Therefore you may make _base_ folder to sync data_pipeline of the two model